lib/temporal/client.rb

```diff
@@ -425,6 +426,17 @@ def query_workflow_executions(namespace, query, filter: {}, next_page_token: nil
       Temporal::Workflow::Executions.new(connection: connection, status: :all, request_options: { namespace: namespace, query: query, next_page_token: next_page_token, max_page_size: max_page_size }.merge(filter))
     end
 
+    # Count the number of workflows matching the provided query
+    # 
+    # @param namespace [String]
+    # @param query [String]
+    #
+    # @return [Temporal::Workflow::CountWorkflowsResult] an integer count of workflows matching the query
+    def count_workflow_executions(namespace, query: nil)
+      response = connection.count_workflow_executions(namespace: namespace, query: query)
+      Workflow::CountWorkflowsResult.new(count: response.count)
```

Comment:
@harsh-stripe Why create a new response object to return here? It'd be simpler to just return the count from this method

Reply:
Made a commit to return the count instead of this wrapped object. I had initially created it thinking it might be useful for a future contributor if they wanted to include aggregation groups based on https://github.com/temporalio/api/blob/master/temporal/api/workflowservice/v1/request_response.proto#L795-L798 in the future, but it isn't required now since it doesn't quite work yet in Temporal.